tools/cell_census_builder/tests/test_builder.py

```diff
@@ -1,26 +1,177 @@
 # General unit tests for cell_census_builder. Intention is to add more fine-grained tests for builder.
 import os
+import unittest
```

Comment:
let's stick with pytest, which is already in use in this repo

Reply:
wholeheartedly agree

If you need any help bootstrapping on pytest, just let us know.  For example, your `generate_h5ad` should be a pytest fixture.

Reply:
I migrated everything to pytest. I think we can improve this further by moving the builder to a class-based approach. Since we have a fair amount of dependencies (ExperimentBuilder, datasets etc), we should be able to implement a relatively clean dependency injection setup using fixtures. I'd defer this work to a separate PR.